src/Vasont.Inspire.Models.csproj

```diff
@@ -20,15 +20,15 @@
     <RepositoryType>git</RepositoryType>
     <PackageTags>CCMS</PackageTags>
     <PackageReleaseNotes>Added SubmissionAttributeModel</PackageReleaseNotes>
-    <Version>1.3.42</Version>
+    <Version>1.3.43</Version>
```

Comment:
This will need to be .44 now that the other PR was completed first.

Reply:
Rebased